src/Mail/LogDriver.php

```diff
@@ -28,9 +28,9 @@ public function __construct(LoggerInterface $logger)
         $this->logger = $logger;
     }
 
-    public function availableSettings(): array
+    public function availableSettings(): object
     {
-        return [];
+        return (object) [];
```

Comment:
What's the benefit of using an object over an array here?

Reply:
We need an object for additional parameters like the type of the form element and also for possible values.

![image](https://user-images.githubusercontent.com/2070323/63895825-91628e00-c9f0-11e9-8da9-42a7d40f1f9d.png)

Reply:
@vingrad Can you expand on that and explain the why? That structure is perfectly valid as an array as well, and it should serialize to JSON in the same way?